CONTRIBUTING.md

```diff
@@ -196,6 +196,15 @@ This allows you to make changes to your local vireo and have them be instantly a
 
 **WARNING** Since the local VireoSDK is symlinked to the JavaScript application be aware of tooling that could accidently delete or modify your local development files unexpectedly.
 
+If you have a .NET application that depends on Vireo you can do the following (you must have nuget.exe available)
+- In the directory with the `VireoSDK.nuspec` file run:
+  ```console
+  nuget pack VireoSDK.nuspec -properties version="[version]"
+  ```
+  
+  replacing \[version\] with some version.
```

Comment:
Instead of slash escaping you can use the back tick \`\[version\]\` so it renders as an identifer `[version]`